src/modules/FioAddress/action.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import type { EdgeCurrencyWallet } from 'edge-core-js'
+
+export const changeFioAddressName = (fioAddressName: string) => ({
```

Comment:
Do not use action creators. Instead directly dispatch the object and create a flow type for the action object.

Reply:
will be refactored